### PR TITLE
Avoid keras 2.6.0

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

The current master branch fails in CI because of the latest release of keras 2.6.0. This PR is a hotfix.

Failed CI example: https://github.com/optuna/optuna/pull/2847/checks?check_run_id=3286012452

## Description of the changes

Introduce the version constraint of keras: `keras<2.6.0`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -146,7 +148,9 @@ def get_extras_require() -> Dict[str, List[str]]:
             "scikit-learn>=0.24.2",
             "scikit-optimize",
             "xgboost",
-            "keras ; python_version<'3.9'",
```
Comment (hvy) [inline setup.py:149]: Just curious but could you comment on the removal of the Python version constraint?

**Review by hvy — COMMENTED** [review]
Thanks for the quick fix. Left a small question but otherwise LGTM.

**Comment by hvy** [comment]
@not522 could you also have a look at this hotfix?

File: setup.py (hunk 1 above)
Comment (nzw0301) [reply setup.py:149]: I removed It accidentally. I'll revert the condition. Thanks!

**Review by hvy — APPROVED** [review]
Thanks for the quick action as always, LGTM!

**Review by not522 — APPROVED** [review]
Thank you, LGTM.

**Pull request merged**